general-superstaq/general_superstaq/superstaq_exceptions_test.py

```diff
@@ -12,31 +12,35 @@
 # limitations under the License.
 # pylint: disable=missing-function-docstring,missing-class-docstring
 
+import textwrap
+
 import general_superstaq as gss
 
 
 def test_superstaq_exception() -> None:
-    ex = gss.SuperstaqException(message="Hello", status_code=500)
-    assert str(ex) == "Status code: 500, Message: 'Hello'"
+    ex = gss.SuperstaqServerException(message="Hello.", status_code=500)
+    assert str(ex) == "Hello. (Status code: 500)"
+    assert ex.message == "Hello. (Status code: 500)"
     assert ex.status_code == 500
-    assert ex.message == "Hello"
-
-
-def test_module_not_found_exception() -> None:
-    ex = gss.SuperstaqModuleNotFoundException("hello_world", "test")
-    assert str(ex) == "Status code: None, Message: ''test' requires module 'hello_world''"
-    assert ex.message == "'test' requires module 'hello_world'"
-
-
-def test_superstaq_not_found_exception() -> None:
-    ex = gss.SuperstaqNotFoundException(message="Where are you")
-    assert str(ex) == "Status code: 404, Message: 'Where are you'"
-    assert ex.status_code == 404
-    assert ex.message == "Where are you"
 
 
 def test_superstaq_unsuccessful_job_exception() -> None:
     ex = gss.SuperstaqUnsuccessfulJobException(job_id="SWE", status="Cancelled")
-    assert str(ex) == "Status code: None, Message: 'Job SWE terminated with status Cancelled.'"
-    assert ex.status_code is None
+    assert str(ex) == "Job SWE terminated with status Cancelled."
     assert ex.message == "Job SWE terminated with status Cancelled."
+
+
+def test_superstaq_server_exception() -> None:
+    ex = gss.SuperstaqServerException(message="This target only supports terminal measurements.")
+    expected = textwrap.fill(
+        textwrap.dedent(
+            """\
+            This target only supports terminal measurements. (Status code: 400, non-retriable error
+            making request to Superstaq API)
+            """
+        ),
+        width=120,
+    )
```

Comment:
nit: the textwrap stuff isn't doing anything here i think
```suggestion
    expected = (
        "This target only supports terminal measurements. (Status code: 400, non-retriable error "
        "making request to Superstaq API)"
    )
```